manager/types/dispatcher.go

```diff
@@ -93,17 +103,20 @@ type NotifyInfo struct {
 	ExecuteMsg string            `json:"execute_msg"`
 }
 
-// Channel types for dispatchers
+// Channel types for dispatchers.
 type Channel string
 
+// the type of channel.
 const (
 	Discord   Channel = "DISCORD"
 	Telegram  Channel = "TELEGRAM"
 	PagerDuty Channel = "PAGERDUTY"
 )
 
+// Reminder is for reminnig alert
```

Comment:
@kim201212 Don't it need to add `.`?

Reply:
@gnongs I fixed it!

[fix annotation](https://github.com/dsrvlabs/vatz/pull/461/commits/852f05d84f854cf341508d46f7dd7c3e2d19e4e2)